apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js

```diff
@@ -10,59 +12,72 @@ const { GET_MY_USERINFO } = require('@condo/domains/user/gql')
-        const admin = await makeLoggedInAdminClient()
-        await User.update(admin, client.user.id, { dv: 1, sender: { dv: 1, fingerprint: 'tests' }, deletedAt: 'true' })
-        await expectToThrowAccessDeniedErrorToResult(async () => {
-            await registerNewServiceUserByTestClient(client)
+        test('Support can create service user', async () => {
+            const [{ email, password }] = await registerNewServiceUserByTestClient(support)
+            const serviceClient = await makeLoggedInClient({ email: email, password: password })
+            const { data: { user } } = await serviceClient.query(GET_MY_USERINFO)
+            expect(user.id).toEqual(serviceClient.user.id)
+            expect(user.type).toEqual(SERVICE)
+        })
+        test('User can not create service user', async () => {
+            await expectToThrowAccessDeniedErrorToResult(async () => {
+                await registerNewServiceUserByTestClient(user)
+            })
+        })
+        test('Deleted support can not register service user', async () => {
+            const anotherSupport = await makeClientWithSupportUser()
+            await User.update(admin, anotherSupport.user.id, { dv: 1, sender: { dv: 1, fingerprint: 'tests' }, deletedAt: dayjs().toISOString() })
+            await expectToThrowAccessDeniedErrorToResult(async () => {
+                await registerNewServiceUserByTestClient(anotherSupport)
+            })
+        })
+        test('Anonymous can not register service user', async () => {
+            await expectToThrowAuthenticationError(async () => {
+                await registerNewServiceUserByTestClient(anonymous)
+            }, 'result')
         })
     })
-    test('anonymous can not register service user', async () => {
-        const client = await makeClient()
-        await expectToThrowAuthenticationError(async () => {
-            await registerNewServiceUserByTestClient(client)
-        }, 'result')
-    })
-})
-
-describe('RegisterNewServiceUserServiceLogic', () => {
-    test('can not register service user with existed email', async () => {
-        const admin = await makeLoggedInAdminClient()
-        const [, userAttrs] = await registerNewServiceUserByTestClient(admin)
-        const email = userAttrs.email
-        // TODO(DOMA-3146): use the GQLError util
-        await catchErrorFrom(async () => {
-            await registerNewServiceUserByTestClient(admin, { email })
-        }, ({ errors }) => {
-            expect(errors[0].originalError.errors[0].data.messages[0]).toEqual(
-                expect.stringContaining(EMAIL_ALREADY_REGISTERED_ERROR))
+    describe('Logic tests', () => {
+        test('can not register service user with existed email', async () => {
+            const [, userAttrs] = await registerNewServiceUserByTestClient(admin)
+            const email = userAttrs.email
+            // TODO(DOMA-3146): use the GQLError util
+            await catchErrorFrom(async () => {
```

Comment:
Maybe fix this? :)

Reply:
It's not throwing GLQError as far as I know